spec.md

```diff
@@ -173,6 +173,164 @@ minimally trusted storage servers is required.
+[Multikey section of the CID spec](https://www.w3.org/TR/cid-1.0/#Multikey)
+as the space `controller`.
+
+When a space is created via an HTTP [POST](#http-api-post-spaces) or
+[PUT](#http-api-put-space-space_id) operation, the controller for that space
+is set, either implicitly or explicitly. 
+
+Implicitly, if no `controller` is specified in the PUT or POST create space
+request, the server MUST determine and set the `controller` from the corresponding
+authorization headers of the request (for example, from the `Authorization` header
+when using HTTP Signatures).
+
+Explicitly, if a client specifies the `controller` as part of the payload
+of the PUT or POST create space request, the server MUST check that the methods
+(key IDs) used in the headers are authorized in the `capabilityInvocation`
+section of the `controller`'s DID document.
+
+See below in the [HTTP POST](#http-api-post-spaces) sections for examples of
+`controller` determination and verification.
+
+Conceptually, the space's controller serves as the root of trust and authorization
+for any operations on the space or its collections or resources.
+That is, any operation requiring an authorization MUST provide a chain of proof
+all the way to the space controller, by one of the following:
+
+1. Direct: Provide a root capability invoked directly by the controller, or
+2. Delegated: Invoke a capability delegated to some other agent by the controller, or
+3. Matching Policy: (if using any kind of access control policy mechanism) Match
+   an authorization policy specified in the `link` property of the space. This
+   resource is related to the space controller because initially, it can only be
+   modified either by the controller or an authorized party delegated to by the
+   controller.
+
+### Performing Authorized API Calls
+
+Unless otherwise explicitly allowed via access control policy (see below),
+all W.A.S. API calls require authorization.
+
+This can be done in one of two ways:
+
+1. (for admin-like root access) Use the `controller` DID directly to sign
+   HTTP API requests using the HTTP Signatures specification.
+2. (for advanced delegatable use cases) Use HTTP Signatures in combination
+   with [Authorization Capabilities v0.3](https://w3c-ccg.github.io/zcap-spec/),
+   and include a capability invocation header in the API request.
+
+### Specifying Access Policy With Space Link Sets
+
+To set access control policy for a space, use the `link` property.
+
+Example (fetching a space's link set):
+
+```http
+GET /space/81246131-69a4-45ab-9bff-9c946b59cf2e/links HTTP/1.1
+Host: example.com
+Accept: application/linkset+json
+Authorization: Signature keyId="did:key:z6MkpBMbMaRSv5nsgifRAwEKvHHoiKDMhiAHShTFNmkJNdVW#z6MkpBMbMaRSv5nsgifRAwEKvHHoiKDMhiAHShTFNmkJNdVW" ...
+```
+
+Response:
+
+```http
+HTTP/1.1 200 OK
+Content-type: application/linkset+json
+
+{
+  "linkset": [
+    {
+      "anchor": "/space/81246131-69a4-45ab-9bff-9c946b59cf2e/",
+      "acl": [
```

Comment:
Don't know if `acl` is the correct name for the relation here. Just using it from one of your earlier examples.

Reply:
> https://solidproject.org/TR/wac#acl-link-relation

But we're not using WAC specifically, are we?